hi.js

```diff
@@ -1,821 +1,821 @@
+settings.mobile.gradSpeed ग्रॅजुअल स्पीड
+settings.mobile.invert इन्वर्ट Y-एक्सिस
+settings.audio.header आवाज़
+settings.audio.sound मास्टर वॉल्यूम
+settings.audio.ambient एम्बियेन्ट वॉल्यूम
+settings.audio.voice वाणी वॉल्यूम
+settings.audio.weps हथियार वॉल्यूम
+settings.audio.player खिलाड़ी की वॉल्यूम
+settings.audio.skin पेहनावे की वॉल्यूम
+settings.audio.ui UI वॉल्यूम
+settings.audio.asset एसेट वॉल्यूम
+settings.audio.dialogue एक्शन वॉल्यूम
+settings.viewmodel.header व्यू मॉडल
+settings.viewmodel.vmSetts व्यू मॉडल सेटिंग्स
+settings.viewmodel.fov FOV
+settings.viewmodel.fps हथियार की FOV
+settings.viewmodel.bobbing हथियार बोब्बिंग
+settings.viewmodel.leaning हथियार लीनिंग
+settings.viewmodel.show.primary चाकू दिखाओ
+settings.viewmodel.show.secondary पिस्तौल दिखाओ
+settings.viewmodel.roundArms गोल हाथ
+settings.viewmodel.show.melee मिली दिखाओ
+settings.viewmodel.show.ads ADS पर हथियार छुपाओ
+settings.viewmodel.toggleAim टॉगल लक्ष्य
+settings.viewmodel.show.hands हाथ दिखाओ
+settings.viewmodel.leftHand बाएं हाथ से
+settings.viewmodel.aimAnim हथियार लक्ष्य एनीमेशन
+settings.viewmodel.rot हथियार रोटेशन
+settings.viewmodel.xOff हथियार X-ऑफसेट
+settings.viewmodel.yOff हथियार Y-ऑफसेट
+settings.viewmodel.zOff हथियार Z-ऑफसेट
+settings.viewmodel.weapSwapY हथियार स्वाप-Y
+settings.viewmodel.weapReloadY हथियार रीलोड Y
+settings.viewmodel.yOffADS हथियार ADS Y-ऑफसेट
+settings.viewmodel.adsFovMlt ADS FOV की शक्ति
+settings.editing.header एडिटिंग
+settings.editing.health.high HUD उच्च हेल्थ
+settings.editing.health.low HUD कम हेल्थ
+settings.editing.speedLines स्पीड लाइंस
+settings.editing.spdLinesCol स्पीड लाइंस रंग
+settings.editing.score.color पॉपअप स्कोर रंग
+settings.editing.score.show पॅापअप स्कोर दिखाओ
+settings.editing.score.scale पॉपअप स्कोर स्केल
+settings.editing.score.shadow पॉपअप स्कोर छाया
+settings.editing.scoreOffX पॉपअप स्कोर X-ऑफसेट
+settings.editing.scoreOffY पॉपअप स्कोर Y-ऑफसेट
+settings.editing.prog.color प्रोग्रेस बार रंग
+settings.editing.prog.opacity प्रोग्रेस बार अपारदर्शिता
+settings.editing.prog.scale प्रोग्रेस बार स्केल
+settings.editing.prog.shadow प्रोग्रेस बार छाया
+settings.editing.progOffX प्रोग्रेस बार X-ऑफसेट
+settings.editing.progOffY प्रोग्रेस बार Y-ऑफसेट
+settings.editing.progRot प्रोग्रेस बार रोटेशन
+settings.editing.progm.color चार्ज बार रंग
+settings.editing.progm.opacity चार्ज बार अपारदर्शिता
+settings.editing.progm.scale चार्ज बार नाप
+settings.editing.progm.shadow चार्ज बार छाया
+settings.editing.progmOffX चार्ज बार X ऑफसेट
+settings.editing.progmOffY चार्ज बार Y ऑफसेट
+settings.editing.progmRot चार्ज बार रोटेश
+settings.editing.saturation सेचुरेशन
+settings.editing.menuSaturation सेचुरेशन (UI)
+settings.editing.hue रंग (खेल में)
+settings.editing.menuHue रंग (UI)
+settings.editing.vignette विग्नेट
+settings.editing.feedLimit किलफीड सीमा
+settings.editing.depth डेप्थ मैप
+settings.editing.green ग्रीन स्क्रीन
+settings.editing.trailcol बुलेट ट्रेल का रंग 
+settings.editing.ropecol grapple  रोप का रंग 

```

Comment:
I'd suggest using Devanagari characters for words like "grapple" as "ग्रेपल" even if they don't have a direct translation in Hindi to keep the language consistent.